common/religion/holy_sites/wc_holy_sites.txt

```diff
@@ -0,0 +1,1326 @@
+		different_culture_opinion = 15
+	}
+}
+lushat_lan = {
+	county = c_lushat_lan
+	
+	character_modifier = {
+		name = holy_site_lushat_lan_effect_name
+		general_opinion = 7
+	}
+}
+
+#################
+#arcane_group
+#################
+### arcane_religion
+dark_portal = {
+	county = c_dark_portal
+	
+	character_modifier = {
+		name = holy_site_dark_portal_effect_name
+		levy_size = 0.05
+		garrison_size = 0.1
+	}
+}
+karazhan = {
+	county = c_karazhan
+	
+	character_modifier = {
+		name = holy_site_karazhan_effect_name
+		intrigue_per_stress_level = 3
+		opinion_of_different_faith = 10
+		learning = 1
+	}
+}
+coldarra = {
+	county = c_coldarra
+	
+	character_modifier = {
+		name = holy_site_coldarra_effect_name
+		taiga_advantage = 5
+		monthly_lifestyle_xp_gain_mult = 0.2
+	}
+}
+dalaran = {
+	county = c_dalaran
+	
+	character_modifier = {
+		name = holy_site_dalaran_effect_name
+		prowess_per_piety_level = 1
+		opinion_of_same_faith = 7
+	}
+}
+#nordrassil
+
+### cult_of_sunwell
+sunwell_plateau = {
+	county = c_sunwell_plateau
+	
+	character_modifier = {
+		name = holy_site_sunwell_plateau_effect_name
+		holy_order_hire_cost_mult = -0.15
+	}
+}
+silvermoon = {
+	county = c_silvermoon
+	
+	character_modifier = {
+		name = holy_site_silvermoon_effect_name
+		archers_damage_mult = 0.25
+		forest_advantage = 7
+	}
+}
+sunstrider_isle = {
+	county = c_sunstrider_isle
+	
+	character_modifier = {
+		name = holy_site_sunstrider_isle_effect_name
+		development_growth_factor = 0.1
+		theocracy_government_vassal_opinion = 10
+	}
+}
+andaroth = {
+	county = c_andaroth
+	
+	character_modifier = {
+		name = holy_site_andaroth_effect_name
+		stewardship = 1
+		monthly_piety_gain_mult = 0.05
+	}
+}
+#dalaran
+#################
+# deathbringer
+#################
+icecrown = {
+	county = c_icecrown
+	
+	character_modifier = {
+		name = holy_site_icecrown_effect_name
+		men_at_arms_limit = 1
+		men_at_arms_cap = 2
+	}
+}
+draktharon = {
+	county = c_draktharon
+	
+	character_modifier = {
+		name = holy_site_draktharon_effect_name
+		dread_gain_mult = 0.3
+		archers_pursuit_mult = 0.1
+	}
+}
+#lordaeron
+#sunwell_plateau
+#dalaran
+
+####################
+# druidism_group
+####################
+boralus = {
+	county = c_boralus
+	
+	character_modifier = {
+		name = holy_site_boralus_effect_name
+		embarkation_cost_mult = -0.2
+		naval_movement_speed_mult = 0.2
+	}
+}
+alterac = {
+	county = c_alterac
+	
+	character_modifier = {
+		name = holy_site_alterac_effect_name
+		mountains_advantage = 8
+		skirmishers_screen_add = 4
+	}
+}
+#lordaeron
+#stromgarde
+#gilneas
+
+### wolf_cult_religion
+blackwald = {
+	county = c_blackwald
+	
+	character_modifier = {
+		name = holy_site_blackwald_effect_name
+		farmlands_advantage = 7
+		stewardship = 1
+		plains_advantage = 6
+	}
+}
+xoojith = {
+	county = c_xoojith
+	
+	character_modifier = {
+		name = holy_site_xoojith_effect_name
+		religious_vassal_opinion = 10
+	}
+}
+#Shrine of Goldrinn
+jillaei = {
```

Comment:
no loc

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/99657760-a1d8ab80-2a6f-11eb-8459-4a3d9ddc3c4c.png)

</details>

Reply:
Done in 4475f275b0c9e0328f98c68f84cf628f01f3fc7e.